AppServices/Validators/ValidationExtensions/ValidationExtensions.cs

```diff
@@ -1,36 +1,34 @@
 ﻿using System;
 using System.Linq;
-using System.Text.RegularExpressions;
 
-namespace AppServices
+namespace Application.Validators.ValidationExtensions
 {
     public static class ValidationExtensions
     {
-        public static bool IsValidFullName(this string completFullName)
+        public static bool IsValidFullname(this string completFullname)
```

Comment:
FullName é o nome completo, seria redundante dizer que é o nome completo
```suggestion
        public static bool IsValidFullname(this string fullname)
```